libsql-server/src/connection/libsql.rs

```diff
@@ -455,6 +455,10 @@ impl<W: Wal> Connection<W> {
 
         let config = config_store.get();
         conn.pragma_update(None, "max_page_count", config.max_db_pages)?;
+        if let Some(mode) = config.durability_mode {
+            tracing::info!("setting PRAGMA synchronous to {:?}", mode);
```

Comment:
```suggestion
            tracing::info!("setting PRAGMA synchronous to {mode:?}");
```
Clippy is gonna complain